### PR TITLE
MGMT-6580 Update bump job to work with REST API & update title

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

File: tools/update_default_release_versions_to_latest.py [hunk 1]
```diff
@@ -460,17 +486,35 @@ def main(args):
...
         github_pr.create_issue_comment(f"Created a PR in app-interface GitLab {gitlab_pr.web_url}")
 
-        github_pr.create_issue_comment(f"Currently no test platform is available, un-holding")
+        github_pr.create_issue_comment(f"Running all tests")
+        github_pr.create_issue_comment(f"/test all")
+
         unhold_pr(github_pr)
 
+
+def get_pr_body(updates_made):
```
Comment (YuviGold) [inline tools/update_default_release_versions_to_latest.py:507]: Please make this info also as the commit message, and not only PR description

File: tools/update_default_release_versions_to_latest.py [hunk 2]
```diff
@@ -460,17 +486,35 @@ def main(args):
...
+    body = " ".join([f"@{user}" for user in PR_MENTION])
+    release_notes = ""
+    for updated_version in updates_made:
```
Comment (YuviGold) [inline tools/update_default_release_versions_to_latest.py:510]: Add details about the bump. For example
4.7.9 -> 4.7.11

File: Dockerfile.build [hunk 3]
```diff
@@ -7,3 +7,6 @@ COPY dev-requirements.txt /tmp/dev-requirements.txt
 RUN pip install -r /tmp/dev-requirements.txt
 
 RUN rm /tmp/*requirements.txt
+
+RUN apt-get update
```
Comment (romfreiman) [inline Dockerfile.build:11]: why do u need to update? It's just a waste of time, no?

Comment (oshercc) [reply Dockerfile.build:11]: Ill remove it

**Pull request closed without merging**